backend/tests/functional_tests/test_user_management.py

```diff
@@ -0,0 +1,115 @@
+"""Tests for user management operations."""
+import json
+from typing import Any
+
+from tests.functional_tests import get_api_client
+from medtagger.api.users.business import set_user_role
+from medtagger.api.auth.business import create_user
+
+example_user_email = 'test@mail.com'
+example_user_password = 'medtagger1'
+example_user_first_name = 'First'
+example_user_last_name = 'Last'
+
+admin_email = 'admin@medtagger.com'
+admin_password = 'medtagger2'
+admin_first_name = 'First'
+admin_last_name = 'Last'
```

Comment:
Please use capital letters: https://www.python.org/dev/peps/pep-0008/#constants

Hmmm... I thing that this should be handled by some linter...

Reply:
Fixed.